canonical/ubuntu.com
# Get Ubuntu meganav section

## Done

- Added Get Ubuntu section to meganav

## QA

- Visit https://ubuntu-com-12221.demos.haus/
- On desktop, click "Get Ubuntu" in the navigation menu (this is the only nav section this PR covers, anything else is being tackled in other PRs)
  - Check that the links work
- View on mobile, see that you can navigate to "Get Ubuntu" via the Menu CTA and interact with all the links
- See that the content matches the design: https://www.figma.com/file/BSP7FqnRMcMvO7kCozJKUa/Ubuntu.com-navigation-update-Iteration-9?node-id=4872%3A27351

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5984
